websites/A/Audible/metadata.json

```
@@ -9,7 +9,8 @@
 		"en": "Audible is an American online audiobook and podcast service that allows users to purchase and stream audiobooks and other forms of spoken word content."
 	},
 	"url": "www.audible.com",
-	"version": "1.0.0",
+	"regExp": "www[.]audible([.][a-z]+)",
+	"version": "1.0.1",
```

Functionality is being added thus minor should be bumped instead
```suggestion
	"version": "1.1.0",
```